.gitignore

```diff
@@ -24,5 +24,8 @@ erl_crash.dump
 .elixir_ls
 Thumbs.db
 
+# asdf
+.tool-versions
```

Comment:
Eh. Thought that this is supposed to be user-specific.

But using this to semi-enforce dev users to use the same tool versions is fine I guess. Removing.

Reply:
This thing actually saved my life quite few times already since I run editor outside of my Docker VM which has different Elixir version than the one in omisego-images/docker-ewallet (and .tool-versions yell at me to install 1.6.5 which is nice…)